model-client/src/commonMain/kotlin/org/modelix/model/metameta/PersistedConcept.kt

```diff
@@ -42,7 +43,8 @@ data class PersistedConcept(val id: Long, val uid: String?) : IConcept, IConcept
     }
 
     override fun getReference(): IConceptReference {
-        throw UnsupportedOperationException()
+        // The reference uses the ID and not the UID because of how MetaModelSynchronizer works.
```

Comment:
The `UID` would be correct. Where exactly is that be a problem in `MetaModelSynchronizer`?

Reply:
I added a longer explanation, why we want the `PersistedConcept#id` be the `ConceptReference#uid` of the created reference.